docs/modules/dev/partials/domain/core-intro.adoc

```diff
@@ -0,0 +1,6 @@
+The xref:dev:domain/core/index.adoc[core component]s of James are:
+
+ * xref:domain/core/server.adoc[Server]
+ * xref:domain/core/mailets.adoc[Mailets]
```

Comment:
A **mailet** is more like a plugin conforming to a standard API, but whose implementation can vary.

In my opinion the component here is more the **mailet container** that provides a "mail processing service".

Reply:
That is normally how I would describe it as well, but I am trying to fit it in the context of the hexagonal architecture.